CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.3.0 September 28, 2015
+
+* Android M support
+* Mapbox support (PR #49)
```

Comment:
we should also include [this change](https://github.com/airbnb/AirMapView/pull/53) in the changelog